pkgs/top-level/all-packages.nix

```diff
@@ -16527,7 +16527,7 @@ in
 
   upower = callPackage ../os-specific/linux/upower { };
 
-  usbguard = libsForQt5.callPackage ../os-specific/linux/usbguard {
+  usbguard = callPackage ../os-specific/linux/usbguard {
     libgcrypt = null;
   };
 
```

Comment:
you also need to kill the `usbguard-nox` entry on  pkgs/top-level/all-packages.nix:16535

Reply:
Good catch, thanks! Added alias to avoid breakage for anyone using `usbguard-nox`.